uint8[3] arrived
uint8[3] img_arrived
